crates/lib3h/src/transport/memory_mock/ghost_transport_memory.rs

```diff
@@ -182,7 +271,7 @@ impl
                         verse.bind(&self.network)
                     };
                     self.maybe_my_address = Some(bound_url.clone());
-
+                    let _ = self.advertise();
```

Comment:
Confusing here- should we check for errors? Or at least give the binding a descriptive name?